README.md

```diff
@@ -614,9 +614,12 @@ Gauges have several modes they can run in, which can be selected with the `multi
 
 - 'all': Default. Return a timeseries per process alive or dead.
 - 'liveall': Return a timeseries per process that is still alive.
+- 'sum': Return a single timeseries that is the sum of the values of all processes, alive or dead.
 - 'livesum': Return a single timeseries that is the sum of the values of alive processes.
 - 'max': Return a single timeseries that is the maximum of the values of all processes, alive or dead.
+- 'livemax': Return a single timeseries that is the maximum of the values of alive processes.
 - 'min': Return a single timeseries that is the minimum of the values of all processes, alive or dead.
+- 'livemin': Return a single timeseries that is the minimum of the values of alive processes.
```

Comment:
since the support is comprehensive with this change, wonder if we should simplify this to be something like:
```md
- 'all': Default. Return a timeseries per process (alive or dead)
- 'sum': Return a single timeseries that is the some of the values of all processes (alive or dead).
- 'max': Return a single timeseries that is the maximum of the values of all processes (alive or dead).
- 'min': Return a single timeseries that is the minimum of the values of all processes (alive or dead).

Prepend 'live' to the beginning of the mode to return the same result but only considering living processes
- eg: 'liveall, 'livesum', 'livemax', 'livemin'
```